_ml-commons-plugin/api.md

```diff
@@ -518,8 +518,24 @@ The API returns the following:
 
 ## Profile
 
-The profile operation returns runtime information on ML tasks and models. The profile operation can help debug issues with models at runtime.
+The profile operation returns runtime information on ML tasks and models. The profile operation can help debug issues with models at runtime. 
```

Comment:
runtime information "about" instead of "on"? "The profile operation can help debug model issues at runtime."